ui/app/serializers/clients/monthly.js

```diff
@@ -36,19 +30,32 @@ export default ApplicationSerializer.extend({
     });
   },
 
+  // For 1.10 release naming changed from 'distinct_entities' to 'entity_clients' and
+  // 'non_entity_tokens' to 'non_entity_clients'
+  // accounting for deprecated API keys here and updating to latest nomenclature
+  homogenizeClientNaming(object) {
```

Comment:
You could extend the activity serializer instead of the base and then you'd have access to this automagically, making it DRYer 😜

Reply:
oh! i like that.